src/styles/card.scss

```diff
@@ -0,0 +1,104 @@
+@import '../styles/variables';
+
+.card {
+  display: flex;
+  flex-direction: column;
+  max-width: 200px;
+  width: 100%;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+
+  &__picture {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+  }
+  &__title {
+    margin-top: 40px;
+  }
+
+  &__title-content {
+    font-family: $main-font-family;
+    font-style: normal;
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    color: $font-main-color;
+  }
+
+  &__product-code {
+    margin-top: 4px;
+
+    &-content {
+      font-family: $main-font-family;
+      font-size: 10px;
+      line-height: 14px;
+      font-weight: 400;
+      color: $font-second-color;
+    }
+  }
+
+  &__stars {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 16px;
+    height: 16px;
+  }
+
+  &__stars-text {
+    font-family: $main-font-family;
+    font-size: 10px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 24px;
+
+    &-title {
+      font-family: $main-font-family;
+      font-weight: 400;
+      font-style: normal;
+      color: $font-second-color;
+      font-size: 12px;
+      line-height: 18px;
+      margin: 0;
+    }
+    &-cost {
+      font-family: $main-font-family;
+      font-weight: 700;
+      color: $font-main-color;
+      font-size: 16px;
+      line-height: 18px;
+      margin: 0;
```

Comment:
```suggestion
```

Reply:
You have normalized file where you remove this margin. You don't need doublicate the code.